### PR TITLE
[ACR] Enable storage account encryption by default

### PR DESCRIPTION
**Pull request by djyou**

### CODE REVIEW

**Comment by djyou** [comment]
@SajayAntony @sivagms

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/template.json [hunk 1]
```diff
@@ -54,7 +54,17 @@
             "sku": {
                 "name": "[parameters('storageAccountType')]"
             },
-            "kind": "Storage"
+            "kind": "Storage",
+            "properties": {
+                "encryption": {
+                    "services": {
+                        "blob": {
+                            "enabled": true
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/template.json]: This isn't so much "enabled by default" as it is forced enabled. Did you want the ability to create the storage account without encryption as a non-default behavior?

**Comment by derekbekoe** [comment]
Will encryption always be available?

**Comment by djyou** [comment]
@derekbekoe The availability of encryption depends on the storage account. Can you elaborate on possible cases where encryption may not be available?

**Comment by derekbekoe** [comment]
@djyou I was wondering if it is available in all regions and on the other Azure clouds but guess I'm thinking too far ahead as ACR is still in preview. Also, the template includes the required storage API version.

**Pull request merged**